libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -376,21 +393,6 @@ export function OutQueueManager(
         executeQueue();
       };
```

Comment:
This `onPostSuccess` callback is now used only in one case – maybe we don't need it anymore? We can directly add the two function calls to the place where needed.